k8s/pkg/apis/longhorn/v1beta2/orphan.go

```diff
@@ -47,6 +47,9 @@ type OrphanSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=v1;v2
 	DataEngine DataEngineType `json:"dataEngine,omitempty"`
+	// Indicates whether the underlying resource can be deleted.
+	// +optional
+	ResourceDeletable bool `json:"resourceDeletable"`
```

Comment:
`ResourceDeletable` sounds like a state. Since this represents an action to be performed, consider renaming it to `DeleteResource` for clarity.

Reply:
Per discussion, this flag is no longer needed. Will be replaced in next PR  by instance process UUID (#3778)